integration/specs/Calendar/calendar-1.spec.js

```diff
@@ -186,4 +186,27 @@ describe('Calendar', () => {
         browser.keys(TAB_KEY);
         expect(calendar.isDayFocused(1)).toBe(true);
     });
+    it('should navigate on the header when using ARROW keys', () => {
+        const calendar = new PageCalendar(CALENDAR);
+        calendar.clickPrevMonthButton();
+        expect(calendar.isPrevMonthButtonFocused()).toBe(true);
+        expect(calendar.isNextMonthButtonFocused()).toBe(false);
+        expect(calendar.isYearSelectFocused()).toBe(false);
+        browser.keys(ARROW_LEFT_KEY);
+        expect(calendar.isPrevMonthButtonFocused()).toBe(true);
+        expect(calendar.isNextMonthButtonFocused()).toBe(false);
+        expect(calendar.isYearSelectFocused()).toBe(false);
+        browser.keys(ARROW_RIGHT_KEY);
+        expect(calendar.isPrevMonthButtonFocused()).toBe(false);
+        expect(calendar.isNextMonthButtonFocused()).toBe(true);
+        expect(calendar.isYearSelectFocused()).toBe(false);
+        browser.keys(ARROW_RIGHT_KEY);
+        expect(calendar.isPrevMonthButtonFocused()).toBe(false);
+        expect(calendar.isNextMonthButtonFocused()).toBe(false);
+        expect(calendar.isYearSelectFocused()).toBe(true);
+        browser.keys(ARROW_RIGHT_KEY);
+        expect(calendar.isPrevMonthButtonFocused()).toBe(false);
+        expect(calendar.isNextMonthButtonFocused()).toBe(false);
+        expect(calendar.isYearSelectFocused()).toBe(true);
+    });
```

Comment:
here I meant only the expects no the entire test, `should move from calendar controls to days when TAB key is pressed` this test is good

Reply:
specifically these expects:
```
expect(calendar.isPrevMonthButtonFocused()).toBe(true);
        expect(calendar.isNextMonthButtonFocused()).toBe(false);
        expect(calendar.isYearSelectFocused()).toBe(false);
```
are the ones that should be removed